pyproject.toml

```diff
@@ -24,57 +27,36 @@ classifiers = [
     "Topic :: Text Processing :: Markup :: HTML",
     "Topic :: Text Processing :: Markup :: reStructuredText",
 ]
+requires-python = ">=3.8.1,<4.0"
+dependencies = [
+    "blinker>=1.4",
+    "docutils>=0.16",
+    "feedgenerator>=1.9",
+    "jinja2>=2.7",
+    "pygments>=2.6",
+    "python-dateutil>=2.8",
+    "rich>=10.1",
+    "unidecode>=1.1",
+    "backports-zoneinfo<1.0.0,>=0.2.1;python_version<'3.9'",
+    "watchfiles",
+]
 
-[tool.poetry.urls]
-"Funding" = "https://donate.getpelican.com/"
-"Tracker" = "https://github.com/getpelican/pelican/issues"
-
-[tool.poetry.dependencies]
-python = ">=3.7,<4.0"
-blinker = ">=1.4"
-docutils = ">=0.16"
-feedgenerator = ">=1.9"
-jinja2 = ">=2.7"
-pygments = ">=2.6"
-python-dateutil = ">=2.8"
-rich = ">=10.1"
-unidecode = ">=1.1"
-markdown = {version = ">=3.1", optional = true}
-backports-zoneinfo = {version = "^0.2.1", python = "<3.9"}
-watchfiles = "^0.19.0"
-
-[tool.poetry.dev-dependencies]
-BeautifulSoup4 = "^4.9"
-jinja2 = "~3.1.2"
-lxml = "^4.3"
-markdown = "~3.4.3"
-typogrify = "^2.0"
-sphinx = "^5.1"
-furo = "2023.03.27"
-livereload = "^2.6"
-psutil = {version = "^5.7", optional = true}
-pygments = "~2.15"
-pytest = "^7.1"
-pytest-cov = "^4.0"
-pytest-sugar = "^0.9.5"
-pytest-xdist = "^2.0"
-ruff = "^0.1.3"
-tox = {version = "^3.13", optional = true}
-flake8 = "^3.8"
-flake8-import-order = "^0.18.1"
-invoke = "^2.0"
-isort = "^5.2"
-black = {version = "^19.10b0", allow-prereleases = true}
+[project.optional-dependencies]
+markdown = ["markdown>=3.1"]
 
-[tool.poetry.extras]
-markdown = ["markdown"]
+[project.urls]
+Homepage = "https://getpelican.com"
+Funding = "https://donate.getpelican.com/"
+"Issue Tracker" = "https://github.com/getpelican/pelican/issues"
+Repository = "https://github.com/getpelican/pelican"
+Documentation = "https://docs.getpelican.com"
 
-[tool.poetry.scripts]
+[project.scripts]
 pelican = "pelican.__main__:main"
 pelican-import = "pelican.tools.pelican_import:main"
-pelican-plugins = "pelican.plugins._utils:list_plugins"
 pelican-quickstart = "pelican.tools.pelican_quickstart:main"
 pelican-themes = "pelican.tools.pelican_themes:main"
+pelican-plugins = "pelican.plugins._utils:list_plugins"
```

Comment:
I'm not sure why this was moved. Perhaps put it back where it was, so it's in alphabetical order?